MobileFilter/sections/specific_web.txt

```diff
@@ -42,6 +42,8 @@ anwap.club,anwap.film,anwap.mobi##body > div[class] > a > strong
 ||mobile.donanimhaber.com/ads^
 !
 turizmekstra.com##img[src^="https://www.turizminsesi.com/d/banner/"]
+||ttf.urzncyno.cn^$image,domain=m.69shu.io
```

Comment:
Usually ads on the sites like that loaded by JS. Please try to find ad script.
And give me HAR file:)